resources/views/components/slideover/partials/header.blade.php

```diff
@@ -13,7 +19,13 @@
                     {{ $title }}
                 </span>
             @endif
-            <label for="{{ $closeId }}" class="absolute right-0 top-1/2 -translate-y-1/2 cursor-pointer text-white">
+            <label
+                @if($idBind)
+                    v-bind:for="{{ $isInForm ? $closeId : $idBind }}"
+                @else
+                    for="{{ $isInForm ? $closeId : $id }}"
+                @endif
```

Comment:
I don't like this repeated multiple times

Reply:
I agree that these aren't great. This could be simplified by putting both values into the v-bind like I had done before with something like

```
v-bind:for="{{ $idBind ?: ('\'' . $id . '\'') }}
```